coinstacks/ethereum/api/src/index.ts

```diff
@@ -4,6 +4,8 @@ import swaggerUi from 'swagger-ui-express'
 import { logger } from '@shapeshiftoss/logger'
 import { middleware } from '../../../common/api/src'
 import { RegisterRoutes } from './routes'
+import swaggerDocument from './swagger.json'
+import { join } from 'path'
```

Comment:
I do try to keep import order as external modules -> scoped packages (@scope/packagename) -> local imports furthest to closest. Might add import order rules to eslint at some point